vue/components/ui/molecules/avatar-list/avatar-list.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+    <div class="avatar-list">
+        <div
+            class="avatar-container"
+            v-bind:class="classes"
+            v-bind:style="{ 'z-index': index }"
+            v-for="(avatar, index) in avatarList"
+            v-bind:key="index"
+        >
+            <div v-if="index == maxElems">
+                {{ remainingElements }}
+            </div>
+            <avatar v-bind="{ size: size, ...avatar }" v-else-if="index < maxElems" />

```

Comment:
Why the `v-else-if`?
Isn't `v-else` enough?

Reply:
It's to not display when `index > maxElems`

Reply:
Oh, I see now, It's doing the for loop to all elements and not limiting the for loop to maxElems.

I would purpose then to limit the for loop and avoid unecessary iterations. Make a computed that limits the array size, or make the for loop run only `maxElems` times to avoid unecessary iterations and simplify this condition